ProcessLib/LiquidFlow/LiquidFlowLocalAssembler-impl.h

```diff
@@ -33,7 +33,7 @@ void LiquidFlowLocalAssembler<ShapeFunction, IntegrationMethod, GlobalDim>::
     const int material_id = _material_properties.getMaterialID(pos);
 
     const Eigen::MatrixXd& permeability = _material_properties.getPermeability(
-        material_id, t, pos, _element.getDimension());
+        material_id, t, pos, _element.getDimension(), 0.0, 0.0);
```

Comment:
what are the magic numbers?

Reply:
Substituted magic numbers by constants initialized with nans.